app/models/camp.rb

```diff
@@ -1,3 +1,16 @@
 class Camp < ApplicationRecord
-  validates :location, presence: true
+  has_many :camps_locations, dependent: :destroy
+  has_many :locations, through: :camps_locations
+  has_many :camps_registrations, dependent: :destroy
+  has_many :users, through: :camps_registrations
+
+  STATUSES = { active: 'true', inactive: 'false' }
```

Comment:
Are we using these statuses? If yes, can we make them `active` / `inactive` instead of true/false